backend/controllers/homepage/homepage.ts

```diff
@@ -230,7 +230,12 @@ const getTasks = ({ activeCaseLoadId, locations, staffId, whereaboutsConfig, key
       href: manageRestrictedPatients.ui_url,
       enabled: () =>
         manageRestrictedPatients.ui_url &&
-        userHasRoles(['SEARCH_RESTRICTED_PATIENT', 'TRANSFER_RESTRICTED_PATIENT', 'REMOVE_RESTRICTED_PATIENT']),
+        userHasRoles([
+          'SEARCH_RESTRICTED_PATIENT',
+          'TRANSFER_RESTRICTED_PATIENT',
+          'REMOVE_RESTRICTED_PATIENT',
+          'RESTRICTED_PATIENT_MIGRATION',
```

Comment:
convention for role codes is to put the service first, hence new role has code different from other restricted patient roles